Vialer/Helpers/VialerStats.swift

```diff
@@ -166,7 +166,26 @@ import Foundation
         
         sendMetrics()
     }
-
+    
+    @objc func incomingCallFailedDeclined(call: VSLCall){
+        initDefaultData()
+        
+        guard !VialerStats.sharedInstance.middlewareUniqueKey.isEmpty else {
+            return
+        }
+        defaultData[VialerStatsConstants.APIKeys.middlewareUniqueKey] = VialerStats.sharedInstance.middlewareUniqueKey
+        
+        setNetworkData()
+        setTransportData()
+        setCallDirection(call.isIncoming)
+        
+        defaultData[VialerStatsConstants.APIKeys.callSetupSuccessful] = "false"
+        defaultData[VialerStatsConstants.APIKeys.failedReason] = "DECLINED"
```

Comment:
I think for there constant is to create a struct with
```
struct FailedReason {
            static let declined: String = "DECLINED"
}
```
I saw there were also others with the failedReason maybe add them also to this struct.